.github/workflows/isotovideo-check-all-test-modules.yml

```diff
@@ -13,4 +13,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Run isotovideo against test code, fail if any test module failed
-        run: podman run --rm -it -v .:/tests:Z --entrypoint '' registry.opensuse.org/devel/openqa/containers/isotovideo:qemu-x86-jq /bin/sh -c 'isotovideo qemu_no_kvm=1 casedir=/tests && jq .result testresults/result-*.json | grep -v ok && echo "Test modules failed" && exit 1'
+        run: |
+          docker run --rm -v .:/tests:Z --entrypoint '' \
```

Comment:
Consider https://github.com/os-autoinst/os-autoinst-distri-example/pull/31#discussion_r1483167930

> Is the switch from podman to docker intended? I thought podman is the preferred way.

Reply:
podman is not provided in the container. I found it easier to just switch to docker. i am not sure if that matters here

Reply:
I did not observe that. See https://github.com/os-autoinst/os-autoinst-distri-example/actions/runs/7275772852/job/19825501448#step:3:1

Reply:
Whops, I will close #35 in favor of yours, it seems we observed the same. 😄